src/components/DragonCard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../style/dragons.css';
+
+const DragonCard = ({ dragon }) => (
+  <div className="dragon-card rounded my-3" key={dragon.id}>
+    <div className="row">
+      <div className="col-sm-12 col-md-5">
+        {dragon.name === 'Dragon 1' ? (
+          <img className="w-100" src={dragon.flickr_images[0]} alt={dragon.name} />
+        ) : (
+          <img className="w-100" src={dragon.flickr_images[0]} alt={dragon.name} />
+        )}
```

Comment:
Hi, @PabloBona this looks the same for me, in both thruthy and falsy condition, please consider updating it, it's a minor issue.

Reply:
Yes you're right; the images have different sizes and at the beginning I wanted to show the dragon 1 the img 2 and vice versa.
But in the end one of the requirements was to show only image 0. And never change it!

Reply:
@hajnaloltyan 
Changes Made It !